test/parallel/test-npm-install.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+const common = require('../common');
+
+const path = require('path');
+const spawn = require('child_process').spawn;
+const assert = require('assert');
+const fs = require('fs');
+
+common.refreshTmpDir();
+
+const npmPath = path.join(
+  common.testDir,
+  '..',
+  'deps',
+  'npm',
+  'bin',
+  'npm-cli.js'
+);
+
+const args = [
+  npmPath,
+  'install'
+];
+
+const pkgContent = '{}';
+
+const pkgPath = path.join(common.tmpDir, 'package.json');
+
+fs.writeFile(pkgPath, pkgContent, testNpm);
```

Comment:
@TheAlphaNerd It's safe to use `writeFileSync()` in parallel tests. The tests in `parallel` work by spawning separate Node processes to run in parallel, so a sync call in one parallel test won't stall or otherwise affect other tests.